docs/release/3.4/release-3-4-12.md

```diff
@@ -0,0 +1,143 @@
+OSG Software Release 3.4.12
+===========================
+
+**Release Date**: 2018-05-10
+
+Summary of changes
+------------------
+
+!!! note
+The repositories referenced by osg-release will be retired on May 15th. Update osg-release before that date.
+
+``` console
+root@host # yum update osg-ca-certs-updater osg-ca-scripts osg-release osg-test rsv
+```
+
+This release contains:
+
+-   Updated all references to grid.iu.edu with opensciencegrid.org
+    -   Package list: osg-ca-certs-updater, osg-ca-scripts, osg-release, osg-test, rsv
```

Comment:
Instead of a comma-separated list, can we make this a markdown list? It's easier to spot packages this way